x-pack/plugins/synthetics/public/apps/synthetics/components/monitor_details/monitor_summary/monitor_complete_sparklines.tsx

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
i think we should move this file into `monitor_history` directory.

Reply:
To be honest I think we should move all the components out of the `monitor_summary` folder since they are shared.

how about `[..]/components/monitor_details/monitor_stats/*.tsx` ?